shared-bindings/touchio/TouchIn.c

```diff
@@ -31,22 +31,29 @@
 //|                print("touched!")"""
 //|
 
-//|     def __init__(self, pin: microcontroller.Pin) -> None:
+//|     def __init__(self, pin: microcontroller.Pin, pull: Optional[digitalio.Pull] = None) -> None:
 //|         """Use the TouchIn on the given pin.
 //|
 //|         :param ~microcontroller.Pin pin: the pin to read from"""
+//|         :param Optional[digitalio.Pull] pull: specify external pull resistor type. If None, assume pull-down or chip-specific implementation
 //|         ...
```

Comment:
Misplaced `"""`. I do this all the time and now prefer to put the closing `"""` on a separate line to make the mistake less likely. Also touched up the arg description a bit.
```suggestion
//|         :param ~microcontroller.Pin pin: the pin to read from
//|         :param Optional[digitalio.Pull] pull: specify external pull resistor type. If None, assume pull-down or chip-specific implementation that does not require a pull.
//|         """
//|         ...
```